process/rating/ratingsData.go

```diff
@@ -0,0 +1,76 @@
+package rating
+
+import (
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+// RatingsData will store information about ratingsComputation
+type RatingsData struct {
+	startRating          uint32
+	maxRating            uint32
+	minRating            uint32
+	metaRatingsStepData  process.RatingsStepHandler
+	shardRatingsStepData process.RatingsStepHandler
+	selectionChances     []process.SelectionChance
+}
+
+// NewRatingsData creates a new RatingsData instance
+func NewRatingsData(settings *config.RatingsConfig) (*RatingsData, error) {
```

Comment:
Configs can be passed as value. One nil check (settings == nil) will no longer be required.

Reply:
fixed